### DISCUSSION TITLE
Able to delete image - Show delete option on hover along with props with image url.

### DISCUSSION THREAD

**Discussion by sar-joshi** [outline/rich-markdown-editor#318] [Ideas]
May it's a good option to put Delete feature for the uploaded image. 

For example, if you hover onto the image, a button/link appears, when clicked an image URL is passed with the props and we could do the necessary later with that URL.

**Comment by cliftonc** [comment]
This was included in: https://github.com/outline/rich-markdown-editor/pull/348
